### PR TITLE
bepasty: 0.5.0 -> 1.1.0

### PR DESCRIPTION
**Pull request by aither64**
###### Description of changes
Follow-up on #116429 to update bepasty to 1.1.0 from 0.5.0, as currently it doesn't work at all, see #116326. Two new
python packages were added as dependencies: xstatic-font-awesome and xstatic-asciinema-player. I've tested it using
the `services.bepasty` module and it seems to work.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/xstatic-asciinema-player/default.nix [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+  src = fetchPypi {
+    inherit version pname;
+    sha256 = "sha256-yA6WC067St82Dm6StaCKdWrRBhmNemswetIO8iodfcw=";
+  };
+
+  # no tests implemented
+  doCheck = false;
+
+  meta = with lib;{
```
Comment (xfnw) [inline pkgs/development/python-modules/xstatic-asciinema-player/default.nix:18]:
```suggestion
  meta = with lib; {
```

**Review by xfnw — CHANGES_REQUESTED** [review]

**Comment by aither64** [comment]
Thanks for taking a look @xfnw, it should be addressed. I've also added one more commit to fix bepasty license, as I've noticed it was wrong.

File: pkgs/development/python-modules/xstatic-asciinema-player/default.nix [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    homepage = "https://github.com/asciinema/asciinema-player";
```
[the remark this answers is not in the document]
Comment (aither64) [reply pkgs/development/python-modules/xstatic-asciinema-player/default.nix]: Yeah, I took homepage links from pypi. xstatic packages are just wrappers of static files taken out of other projects... so I'm not sure. I've changed it to package sources as you've found them.

**Comment by makefu** [comment]
thanks for putting in the effort to update the package!

**Comment by makefu** [comment]
tested with nixpkgs-review the bepasty-server, lgtm!

**Review by makefu — APPROVED** [review]

**Pull request merged**